### PR TITLE
[OPENJDK-2199] Update tests URI to refer to new application source repository

### PR DESCRIPTION
**Pull request by jhuttana**
JIRA ticket: https://issues.redhat.com/browse/OPENJDK-2199
Update tests in ubi9 branch to refer to new application source repo.
Locally verified the tests after updating URI and they are passing.

### CODE REVIEW

**Comment by jmtd** [comment]
Thanks for picking this over to ubi9. Looks good, I'm just running the tests locally.

**Review by jmtd — APPROVED** [review]

**Comment by jmtd** [comment]
Tests are fine. I'm waiting until current images ship then I can merge.

**Comment by jhuttana** [comment]
Thank you Jon :)

**Pull request merged**
